From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Sheludko <ishell@chromium.org>
Date: Tue, 15 Oct 2019 10:46:52 +0200
Subject: [ptr-compr] Temporarily disable 31 bit Smis on 64-bit architectures

The reason is to unblock M79 blocked by chromium:1009439 while full
solution is not ready yet.

This CL will be reverted after the M79 branch point.

Bug: v8:9767, chromium:1009439
Change-Id: I5302d86fe953ecd94d9a4bba0d29c807b7b9d703
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/1862554
Reviewed-by: Toon Verwaest <verwaest@chromium.org>
Commit-Queue: Igor Sheludko <ishell@chromium.org>
Cr-Commit-Position: refs/heads/master@{#64286}

diff --git a/BUILD.gn b/BUILD.gn
index 6207d71c542c46a4716a51baeb8518eca0f8e1c0..db7179968237406318c64d68eae08027860e98f9 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -108,7 +108,7 @@ declare_args() {
 
   # Enable pointer compression (sets -dV8_COMPRESS_POINTERS).
   v8_enable_pointer_compression = false
-  v8_enable_31bit_smis_on_64bit_arch = true
+  v8_enable_31bit_smis_on_64bit_arch = false
 
   # Sets -dOBJECT_PRINT.
   v8_enable_object_print = ""
